cpp/ql/src/Metrics/Internal/IncludeResolutionStatus.ql

```diff
@@ -0,0 +1,19 @@
+/**
+ * @name Count of includes that failed to be resolved as well as compilation error due to undefined token
+ * @description A count of all failed includes and compilation errors due to undefined tokens.
+ * @kind table
+ * @tags summary telemetry include resolution
```

Comment:
Note that tags must come from a specific set: https://codeql.github.com/docs/writing-codeql-queries/metadata-for-codeql-queries/
```suggestion
```

Reply:
Removed